src/components/drawing/symbol_defs.js

```diff
@@ -1,7 +1,10 @@
 'use strict';
 
 var parseSvgPath = require('parse-svg-path');
-var round = require('@plotly/d3').round;
+var round = // require('@plotly/d3').round;
```

Comment:
@birkskyum I noticed requirejs somehow get confused with this function we added to our fork in https://github.com/d3/d3/commit/ddf0fe85f0d30887637bf73881f6a5ce37b4644e 
So I decided to simply inline it here.
I tried to revert the change once the all the `publish-dist` tests pass; but it didn't work. See https://app.circleci.com/pipelines/github/plotly/plotly.js/11179/workflows/d58363bf-53e6-48b5-bf0f-b3fb77e03570/jobs/247284/parallel-runs/0/steps/0-115
So I apply this change again. And it solved our problem.